pkgs/applications/networking/mkchromecast/default.nix

```diff
@@ -0,0 +1,73 @@
+{ pkgs, stdenv, fetchurl, python3Packages, python3, makeWrapper
+, alsaSupport ? false
+, systemTray ? true
+, ffmpegSupport ? true
+, youtubeSupport ? true
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mkchromecast";
+  version = "0.3.8.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/muammar/${pname}/archive/${version}.tar.gz";
+    sha256 = "1clr02zphpkjnnw5l1mgbjsk5ycqryajnlpb8wfdb5ax72ca7066";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = with python3Packages; [
+    PyChromecast
+    psutil
+    mutagen
+    flask
+    netifaces
+    requests
+    pkgs.vorbis-tools
+    pkgs.sox
+    pkgs.flac
+    pkgs.faac
+    pkgs.lame
+    pkgs.ffmpeg
+    pkgs.nodejs
```

Comment:
Dependencies that aren't Python packages should be in `buildInputs` and they should be specified as arguments. However, you can probably leave them out anyway because you need to patch commands such as https://github.com/muammar/mkchromecast/blob/0.3.8.1/mkchromecast/video.py#L61 using `substituteAll`.

Reply:
Actually no need to patch. We can just use `makeWrapper` to add these packages to `$PATH`.

Reply:
As in another `--prefix` with `--prefix PATH : "$ffmpeg.outPath/bin:$PATH"`? What's the best way to go about doing this?